src/main/java/reposense/git/GitDiff.java

```diff
@@ -18,4 +23,14 @@ public static String diffCommit(String root, String lastCommitHash) {
         Path rootPath = Paths.get(root);
         return runCommand(rootPath, "git diff -U0 " + lastCommitHash);
     }
+
+    /**
+     * Returns a list of committed files with the corresponding number of lines added and deleted in the repo
+     * {@code repoRoot}.
+     */
+    public static List<String> getModifiedFiles(Path repoRoot) {
```

Comment:
Should also follow Eugene's suggestion to standardize to `FilesList` -> `getModifiedFilesList`